instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+
+class Login extends React.Component {
+    constructor(){
+        super();
+        this.setState = {}
+    }
+    loggedIn = (e) => {
+        e.preventDefault();
+        const user = document.getElementById("user").value;
+        console.log(user.value)
+        localStorage.setItem('user', user)
```

Comment:
Since we want to make the page reload upon logging in, `e.preventDefault()` will not be necessary here (considering that the default behavior of `onSubmit` events is to refresh the page).